packages/expo-video-thumbnails/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "expo-video-thumbnails",
+  "version": "1.0.0",
+  "description": "ExpoVideoThumbnails standalone module",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {},
+  "keywords": [
+    "react-native",
+    "expo",
+    "expo-video-thumbnails"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://github.com/expo/expo/tree/master/packages/expo-video-thumbnails",
+  "peerDependencies": {
+    "@unimodules/core": "~1.0.0"
+  },
+  "gitHead": "8413e821076a2eca36b302e68dd628ce2e1f591a"
+
+,"_resolved": ""
+,"_integrity": ""
+,"_from": "expo-video-thumbnails@4.0.0"
```

Comment:
Could you please remove those `_…` fields and also remove `.gitkeep` from `src`?